internal/pkg/levee/levee.go

```diff
@@ -57,17 +57,28 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	for fn, sources := range sourcesMap {
 		for _, b := range fn.Blocks {
 			for _, instr := range b.Instrs {
-				v, ok := instr.(*ssa.Call)
-				if !ok {
-					continue
-				}
+				switch v := instr.(type) {
 
-				callee := v.Call.StaticCallee()
-				switch {
-				case fieldPropagators.IsFieldPropagator(v):
-					propagations[v] = propagation.Dfs(v, conf, taggedFields)
-					sources = append(sources, source.New(v))
-				case callee != nil && conf.IsSink(utils.DecomposeFunction(callee)):
+				case *ssa.Call:
+					callee := v.Call.StaticCallee()
+					switch {
+					case fieldPropagators.IsFieldPropagator(v):
```

Comment:
This code has changed recently. Travis seems happy, but you may want to merge master to reduce the diff.

Reply:
It does have @PurelyApplied's changes from #221

Reply:
Oh. Sorry about that. I was thinking of #223, but that hasn't been merged yet.

Reply:
Michael might be thinking of (pending) #223, which moves that `IsFieldPropagator` logic into `source`.  No matter which gets merged first, the conflicts should be easy to resolve.